test/test_van/test_email.py

```diff
@@ -81,6 +189,15 @@ def assert_matching_tables(table1, table2, ignore_headers=False):
     },
 ]
 
+mock_response_enriched = deepcopy(mock_response)
+mock_response_enriched[0]["emailMessageContent"] = sample_content_single
+mock_response_enriched[1]["emailMessageContent"] = sample_content_single
+mock_response_enriched[2]["emailMessageContent"] = sample_content_single
+mock_response_enriched[3]["emailMessageContent"] = sample_content_single
+mock_response_enriched[4]["emailMessageContent"] = sample_content_full
+
+print(mock_response_enriched)
```

Comment:
Is there a reason we're printing something here, or is it left over from when you were debugging?

Reply:
That was probably a holdover from testing things in a notebook. Will delete. The data structures I got directly from @matthewkrausse , just with a bit of field value changing (so it didn't have GGT data in it).